src/NHibernate.Test/Operations/OneToOne.hbm.xml

```diff
@@ -32,10 +32,12 @@
 
 	<class name="PersonalDetails" table="OPS_PERS_DETAIL">
 		<id name="Id" column="ID" type="long">
-			<generator class="increment"/>
+			<generator class="foreign">
+				<param name="property">Person</param>
+			</generator>
```

Comment:
We already had a test really near what is required for reproducing #2608, unfortunately ignored and anyway, not using an id strategy on its parent which would be adequate for triggering the bug. As switching the parent id strategy would impact other tests, I have just fix that other test.
Strictly speaking it should go in a dedicated PR, but let it be here.